### DISCUSSION TITLE
Mapsui in Avalonia Cross-Platform Application

### DISCUSSION THREAD

**Discussion by rrangelo** [Mapsui/Mapsui#2307] [Q&A]
Hi all!

How can I use Mapsui in a Cross-Platform application with Avalonia?

I have 5 projects: Core, Desktop, Browser, Android and iOS. How can I configure Mapsui if my views are the same for all of them?

Thanks in advanced!

**Comment by pauldendulk** [comment]
The MapControl.Map object holds everything you need. You can create a Map object in your shared code and assign it to the MapControl.Map of the platform.
